RELNOTES.md

```diff
@@ -0,0 +1,23 @@
+# API Changes
+
+
+# Behavior Changes
+
+
+# New Features
+
+* xds: add support for aggregate clusters (#4332)
+
+# Performance Improvements
+
+* transport: remove decodeState to reduce allocations (#3313, #4423)
+  - Special thanks: @JNProtzman
+
+# Bug Fixes
+
+* xds_client/rds: weighted_cluster totalWeight default to 100 (#4439)
+  - Special thanks: @alpha-baby
+* transport: unblock read throttling when controlbuf exits (#4447)
```

Comment:
With this, every backport will needs 2 PRs?
One for the real change on the release branch, one to remove the note from master

Reply:
I have potential solutions for this and the other two problems discussed earlier, by post-processing RELNOTES.md before using as the actual release notes:

1. Automatically add PR numbers in parenthesis (the PR# that added the line via `git blame`)
2. Automatically add "Special Thanks" if the author was not one of us
3. Remove lines committed in PRs that have "no release notes" label on them.  This label could be added post-merge if either: 1. we change our minds about including the PR in the release notes, or 2. the PR is cherry-picked

1 & 2 can be done using the `git` CLI tool alone.  3 would need to talk to github.

Thoughts?

FWIW the idea of keeping release notes in revision control was inspired by: https://github.com/envoyproxy/envoy/blob/main/docs/root/version_history/current.rst

Reply:
It's still not clear to me how we can do these
- for 1&2, a script similar to `vet.sh` that has to run for each PR?
- 3 will need to make PRs, who will make it? A bot or one of us? And we do that when we make the release?

Reply:
For all 3, it would just be a 1-off script that we run when it's time to do the release.  It doesn't need to update the committed file.

I think we could delete the notes after every release.